script/docker_ci.sh

```diff
@@ -57,9 +65,13 @@ if [ ! -z "${ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE+x}" ]
 then
     ALPAKA_DOCKER_ENV_LIST+=("--env" "ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE=${ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE}")
 fi
-if [ ! -z "${ALPAKA_ACC_CPU_BT_OMP4_ENABLE+x}" ]
+if [ ! -z "${ALPAKA_ACC_ANY_BT_OMP5_ENABLE+x}" ]
+then
+    ALPAKA_DOCKER_ENV_LIST+=("--env" "ALPAKA_ACC_ANY_BT_OMP5_ENABLE=${ALPAKA_ACC_ANY_BT_OMP5_ENABLE}")
+fi
+if [ ! -z "${ALPAKA_OFFLOAD_MAX_BLOCK_SIZE+x}" ]
 then
-    ALPAKA_DOCKER_ENV_LIST+=("--env" "ALPAKA_ACC_CPU_BT_OMP4_ENABLE=${ALPAKA_ACC_CPU_BT_OMP4_ENABLE}")
+    ALPAKA_DOCKER_ENV_LIST+=("--env" "ALPAKA_OFFLOAD_MAX_BLOCK_SIZE=${ALPAKA_OFFLOAD_MAX_BLOCK_SIZE}")
```

Comment:
Is `ALPAKA_OFFLOAD_MAX_BLOCK_SIZE` limited to OpenMP for now?
Should we rename it to `ALPAKA_OMP_OFFLOAD_MAX_BLOCK_SIZE`

Reply:
It is also used by OpenAcc.